### PR TITLE
Update AutoGenerated Password

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Change how moderating participant password is created to include a capitalized letter
* Remove space inside string interpolation

[#98987112]

### CODE REVIEW

File: app/models/think_feel_do_dashboard/concerns/group.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@ def create_moderating_participant
       end
 
       def create_participant(study_id)
-        password = SecureRandom.hex(64)
+        password = SecureRandom.hex(32) + SecureRandom.hex(32).upcase
```
Comment (eschlange) [inline app/models/think_feel_do_dashboard/concerns/group.rb]: What is the purpose of this change?

Comment (michaelwehrley) [inline app/models/think_feel_do_dashboard/concerns/group.rb]: Good question.  `SecureRandom.hex(32)` only returns lowercase characters and whole numbers.  The password changes that are now require need at least one uppercase letter. :smile:

**Comment by eschlange** [comment]
:eyes: :+1:

File: app/models/think_feel_do_dashboard/concerns/group.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/models/think_feel_do_dashboard/concerns/group.rb]: Another solution - and more likely - I think a better one, would have been to use `base64` instead of `hex`.
